src/data/shared/case-studies/case-studies-list.js

```diff
@@ -561,8 +562,7 @@ const caseStudiesData = {
     {
       title: 'Sysdig',
       logo: Sysdig,
-      description:
-        '<strong>Sysdig</strong> uses eBPF to enhance the performance of Falco, their open-source runtime security project',
+      description: '<strong>Sysdig</strong> uses eBPF through falco for real-time threat detection',
```

Comment:
Can you read https://sysdig.com/blog/sysdig-and-falco-now-powered-by-ebpf/ and summarize it for the description? Please also add it as a blog

Reply:
done